src/org/labkey/test/components/react/BaseReactSelect.java

```diff
@@ -339,85 +394,64 @@ protected ElementCache newElementCache()
         @NotNull
         WebElement findOption(String option)
         {
-            Locator loc = Locators.options.withText(option);
-            return loc.findElement(selectMenu);
+            return Locators.options.withText(option).findElement(selectMenu);
         }
     }
 
     public static abstract class Locators
     {
```

Comment:
as an informational note, we intend to discontinue the pattern of having static Locators classes in our components- where it's practicable, it's probably best to put locators in the appropriate elementCache or make them class members if there's a need to reference them on a static basis